staging/src/k8s.io/apimachinery/pkg/apimachinery/announced/group_factory.go

```diff
@@ -42,10 +42,16 @@ type GroupVersionFactoryArgs struct {
 
 // GroupMetaFactoryArgs contains the group-level args of a GroupMetaFactory.
 type GroupMetaFactoryArgs struct {
+	// GroupName is the name of the API-Group
```

Comment:
I wanted to write 'fully-qualified' or some such, but I don't see any consistency with existing apis.

```
λ rg 'const GroupName'
abac/register.go
26:const GroupName = "abac.authorization.kubernetes.io"

abac/v1beta1/register.go
25:const GroupName = "abac.authorization.kubernetes.io"

abac/v0/register.go
25:const GroupName = "abac.authorization.kubernetes.io"

apps/register.go
30:const GroupName = "apps"

apps/v1beta1/#register.go#
28:const GroupName = "apps"

apps/v1beta1/register.go
26:const GroupName = "apps"

authentication/#register.go#
27:const GroupName = "authentication.k8s.io"

authentication/register.go
25:const GroupName = "authentication.k8s.io"

authentication/v1beta1/register.go
26:const GroupName = "authentication.k8s.io"

authorization/register.go
25:const GroupName = "authorization.k8s.io"

authorization/v1beta1/register.go
26:const GroupName = "authorization.k8s.io"

autoscaling/register.go
25:const GroupName = "autoscaling"

autoscaling/v1/register.go
26:const GroupName = "autoscaling"

batch/register.go
25:const GroupName = "batch"

batch/v1/register.go
26:const GroupName = "batch"

batch/v2alpha1/register.go
26:const GroupName = "batch"

certificates/register.go
30:const GroupName = "certificates.k8s.io"

certificates/v1beta1/register.go
26:const GroupName = "certificates.k8s.io"

componentconfig/register.go
30:const GroupName = "componentconfig"

componentconfig/v1alpha1/register.go
25:const GroupName = "componentconfig"

extensions/register.go
25:const GroupName = "extensions"

extensions/v1beta1/register.go
26:const GroupName = "extensions"

imagepolicy/register.go
25:const GroupName = "imagepolicy.k8s.io"

imagepolicy/v1alpha1/register.go
26:const GroupName = "imagepolicy.k8s.io"

policy/register.go
25:const GroupName = "policy"

policy/v1alpha1/register.go
26:const GroupName = "policy"

policy/v1beta1/register.go
26:const GroupName = "policy"

rbac/register.go
25:const GroupName = "rbac.authorization.k8s.io"

rbac/v1alpha1/register.go
25:const GroupName = "rbac.authorization.k8s.io"

rbac/v1beta1/register.go
25:const GroupName = "rbac.authorization.k8s.io"

storage/register.go
25:const GroupName = "storage.k8s.io"

storage/v1beta1/register.go
26:const GroupName = "storage.k8s.io
```